css/css-pseudo/marker-variable-local.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
```

Comment:
Can we name this file `-computed-style.html` instead of `-local.html`?

Reply:
done.